packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -466,15 +465,17 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
         if (props.multiselectable) {
             return null;
         }
-        const labelledBy: string = `${this.props.labelledBy} ${triggerId}`;
+        const labelledByFromProp: string = isNil(this.props.labelledBy)
+            ? ""
+            : `${this.props.labelledBy} `;
+        const labelledBy: string = `${labelledByFromProp}${triggerId}`;
```

Comment:
This could just be:
```
const labelledBy: string = `${isNil(this.props.labelledBy) ? "" : `${this.props.labelledBy} `}${triggerId}`;
```

To prevent adding another const. Same for the one below.

Reply:
sure